src/hooks/postrun/plugin-postrun.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+const { logger } = require('@twilio/cli-core').services.logging;
+const chalk = require('chalk');
+
+const AUTOCOMLETE_WARNING = `If you’re using autocomplete, you’ll need to run ${chalk.bold(
+  'twilio autocomplete',
+)} after installing a plugin and then open a new terminal window. The CLI needs to re-build its cache.`;
+const AUTOCOMLETE_ALERT = `If you are running bash or zsh on macOS or Linux, you can run one of the two commands below (as appropriate for the shell you are using): ${chalk.bold(
```

Comment:
nit: the statement says `you can run one of the two commands below (as appropriate for the shell you are using)` but they are shown in the same line (as per the screenshot you shared), maybe we can add `\n` to have the commands show up in the next line.

Reply:
Formatted it